[DOCS] Event filters and Host isolation exceptions by Policy (#1615)
elastic/security-docs: docs/getting-started/configure-integration-policy.asciidoc

[[configure-endpoint-integration-policy]]
= Configure the {endpoint-sec} integration policy

> After the {agent} is installed with the {endpoint-sec} integration, several protections features — including preventions against malware, ransomware, memory threats, and malicious behavior — are automatically enabled on protected hosts (a Platinum or Enterprise license feature). If needed, you can update the integration policy to configure protection settings, event collection, antivirus settings, trusted applications, event filters, and host isolation exceptions to meet your organization's security needs.

Reviewer: `are automatically enabled on protected hosts (a Platinum or Enterprise license feature)`

We should maybe move or delete the note about Platinum or Enterprise, since not all protections in a policy are platinum/enterprise (Ransomware, Memory Threat, and Malicious Behavior are the paid protections)

Author: @caitlinbetz How about a revision? I'm hesitant to delete it, since otherwise the reader might not realize some of these features aren't free. We're pretty clear about licensing when we discuss each feature individually further down the page, but a blanket statement at the top might help set expectations.
```suggestion
After the {agent} is installed with the {endpoint-sec} integration, several protections features — including preventions against malware, ransomware, memory threats, and malicious behavior — are automatically enabled on protected hosts (some features require a Platinum or Enterprise license). If needed, you can update the integration policy to configure protection settings, event collection, antivirus settings, trusted applications, event filters, and host isolation exceptions to meet your organization's security needs.
```

Reviewer: Yeah agreed! Just worried that it was confusing before. This looks good!

> NOTE: You must have admin permissions in {kib} to configure an integration policy.

Reviewer: We may want to clarify this; this section implies the user will be editing the policy in the security application, which currently requires _superuser_.

Configuring integrations within Fleet requires Fleet Admin

Author: @caitlinbetz Do you think we need to mention the Fleet requirement here? This page only covers configuring a policy from the POV of the Security app, so mentioning Fleet permissions here might be distracting. We do cover installing and configuring via Fleet/Integrations [elsewhere](https://www.elastic.co/guide/en/security/current/install-endpoint.html).
```suggestion
NOTE: You must have the {kib} `superuser` role to configure an integration policy in the {security-app}.
```

* <<adv-policy-settings>>
* <<save-policy>>

> 4. Click the **Trusted applications**, **Event filters**, and **Host isolation exceptions** tabs to review the endpoint policy artifacts assigned to this integration policy (for more information, refer to <<trusted-apps-ov>>, <<event-filters>>, and <<host-isolation-exceptions>>). On each tab, you can:

Author: @benironside I think "and" is appropriate because we want the user to click on _all_ of these tabs if they need to -- I just didn't want to have three identical steps to cover all three tabs. We could change "each tab" to plural to match; I don't think it's absolutely necessary but it works this way too:
```suggestion
4. Click the **Trusted applications**, **Event filters**, and **Host isolation exceptions** tabs to review the endpoint policy artifacts assigned to this integration policy (for more information, refer to <<trusted-apps-ov>>, <<event-filters>>, and <<host-isolation-exceptions>>). On these tabs, you can:
```

* Expand and view an artifact — Click the arrow next to its name.

> * View an artifact's full details — Click the actions button (**...**), then select **View full details**.

Reviewer 2: Does "View artifact details" or "View an artifact's details" work here? Not sure that you need to use "full" twice in this line.

Author: ```suggestion
* View an artifact's details — Click the actions button (**...**), then select **View full details**.
```

> * View full details for an artifact — Click the actions button (**...**), then select **View full details**.

Reviewer 3: ```suggestion
* View an artifact's full details — Click the actions button (**...**), then select **View full details**.
```
just putting it out there. I always wonder about "for" vs "of" etc in cases like this, this phrasing avoids having to choose.

Reviewer 3: alternatively, what about "view full artifact details"

Reviewer 3: Could we remove "an" in "an artifact" from all of these bullets?

Author: @benironside Your first revision looks good to me. I think removing "an" in all the bullets would feel a little clipped.

* Unassign an artifact (Platinum or Enterprise subscription) — Click the actions button (**...**), then select **Remove from policy**. This does not delete the artifact; this just unassigns it from the current policy.
* Assign an existing artifact (Platinum or Enterprise subscription) — Click **Assign _x_ to policy**, then select an item from the flyout. This view lists any existing artifacts that aren't already assigned to the current policy.

> NOTE: You can't create new endpoint policy artifact while configuring an integration policy. To create a new artifact, go to its main page in the {security-app} (for example, **Manage** -> **Trusted applications** to add a new trusted application).

Reviewer 3: ```suggestion
NOTE: You can't create a new endpoint policy artifact while configuring an integration policy. To create a new artifact, go to its main page in the {security-app} (for example to create a new trusted application, go to **Manage** -> **Trusted applications**).
```
